style.css

```diff
@@ -76,4 +75,224 @@ body {
   .social-icons ul li a {
     text-decoration: none;
   }
+
+  .projects {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: auto;
+  }
+
+  .projects h2 {
+    text-align: center;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    color: #172b4d;
+    position: relative;
+  }
+
+  .projects .line {
+    display: block;
+    height: 4px;
+    width: 48px;
+    border-radius: 24px;
+    background-color: #36b37e;
+  }
+
+  .project-cards {
+    margin-top: 60px;
+    position: relative;
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    gap: 20px;
+  }
+
+  .project-card {
+    width: 327px;
+    height: 474px;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
> Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels

- [ ] According to the project requirement, it suggests you use percentages instead of px for your **widths**, this helps make your portfolio adapt to different screen sizes, In this case, this will make your work card adapt to multiple screens sizes easily, and look professional and give the best user experience.

Reply:
Can I work with rems and ems in this case @V-Blaze